repos/re-theme/container/Dockerfile

```diff
@@ -1,88 +1,39 @@
-# Allows overwriting where the base image is pulled from
-# Must come before the FROM directive
-ARG KEG_NODE_VERSION
-ARG GIT_STAGE_IMAGE_FROM=node:$KEG_NODE_VERSION
-ARG KEG_IMAGE_FROM=keg-base:latest
-FROM $KEG_IMAGE_FROM as builder
-
-WORKDIR /
-
-# Add npm-run-all globally
-RUN yarn global add npm-run-all
-
-# Should we use the local copy of the tap repo when building
-ARG KEG_COPY_LOCAL
-
-# Path of the app within the docker container
-ARG DOC_APP_PATH=/keg/tap
-
-# Copy over the app to a temp directory
-COPY . /keg-temp/
-
-# Update the build steps git config to include the key
-# Pull down the tap locally if a git tap url exists
-# Otherwise copy over the local version from keg-temp
-RUN git config --global url.https://$PUBLIC_GIT_KEY@github.com/.insteadOf https://github.com/; \
-    if [ -z "$KEG_COPY_LOCAL" ] && [ "$GIT_APP_URL" != "INITIAL" ]; then \
-      git clone $GIT_APP_URL $DOC_APP_PATH; \
-    else \
-      cp -R /keg-temp/ $DOC_APP_PATH; \
-    fi; \
-    rm -rf /keg-temp
-
-# Copy over the package.json, and yarn.lock files
-COPY package.json $DOC_APP_PATH/package.json
-COPY *.lock $DOC_APP_PATH/
-
-# Install the dependecies with yarn setup, then remove the .npmrc
-RUN cd $DOC_APP_PATH; \
-    yarn install --ignore-engines; \
-    yarn cache clean; \
-    rm -f .npmrc
+# Path of the tap within the docker container
```

Comment:
* comment not related to this file*

I don't believe this was an issue before but: 
when I run `keg retheme start --sync jsutils:install:start`
the app doesn't start (because it's attached and watching for jsutils changes?)

maybe we need to have the `retheme` jsutils sync start to be detached? like in the components
- [keg-components ref](https://github.com/simpleviewinc/keg-cli/blob/548e14da84ae79eb353bafab5817993d6b87b881/containers/components/mutagen.yml#L60)
- [re-theme ref](https://github.com/simpleviewinc/keg-hub/blob/31d390b7f5873f604ce45b70d19f9cc993e32288/repos/re-theme/container/mutagen.yml#L40)

* I tried adding `detach: true` to `retheme` sync jsutils and got this error
* [image](https://user-images.githubusercontent.com/3317835/106654593-8dc42680-6555-11eb-9654-e7612a3f62c6.png)